### PR TITLE
VDS-3089: Update csvhelper and attempt to fix breaking changes

### PR DESCRIPTION
**Pull request by pushnitsa**
Description: 

--

QA-test: 

Demo-test: 

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.DescriptionExportImport_1.0.0-pr-13-774d7da8.zip

### CODE REVIEW

File: src/VirtoCommerce.DescriptionExportImportModule.Core/Models/ImportConfiguration.cs [hunk 1]
```diff
@@ -1,21 +1,20 @@
-using System;
 using System.Globalization;
-using CsvHelper;
 using CsvHelper.Configuration;
 
 namespace VirtoCommerce.DescriptionExportImportModule.Core.Models
 {
-    public sealed class ImportConfiguration : Configuration
+    public static class ImportConfiguration
```
Comment (trueboroda) [inline src/VirtoCommerce.DescriptionExportImportModule.Core/Models/ImportConfiguration.cs]: Rename to smth like this: CsvConfigurationFactory

**Review by trueboroda — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DescriptionExportImportModule.Core/Models/ImportConfiguration.cs [hunk 2]
```diff
@@ -1,21 +1,20 @@
-using System;
 using System.Globalization;
-using CsvHelper;
 using CsvHelper.Configuration;
 
 namespace VirtoCommerce.DescriptionExportImportModule.Core.Models
 {
-    public sealed class ImportConfiguration : Configuration
+    public class ImportConfigurationFactory
```
Comment (asvishnyakov) [inline src/VirtoCommerce.DescriptionExportImportModule.Core/Models/ImportConfiguration.cs]: Sealed, no virtual 'cause we don't allow extend module

File: src/VirtoCommerce.DescriptionExportImportModule.Data/Services/CsvImportReporter.cs [hunk 3]
```diff
@@ -59,8 +59,7 @@ public async ValueTask DisposeAsync()
         {
             using (await AsyncLock.GetLockByKey(_filePath).LockAsync())
             {
-                await _streamWriter.FlushAsync();
-                _streamWriter.Close();
+                await _streamWriter.DisposeAsync();
```
Comment (asvishnyakov) [inline src/VirtoCommerce.DescriptionExportImportModule.Data/Services/CsvImportReporter.cs:62]: Why Flush was removed?

File: src/VirtoCommerce.DescriptionExportImportModule.Data/Services/CsvPagedDataImporter.cs [hunk 4]
```diff
@@ -50,6 +55,8 @@ public virtual async Task ImportAsync(ImportDataRequest request, Action<ImportPr
 
             var importProgress = new ImportProgressInfo { Description = "Import has started" };
 
+            SetupErrorHandlers(progressCallback, configuration, errorsContext, importProgress, importReporter);
```
Comment (asvishnyakov) [inline src/VirtoCommerce.DescriptionExportImportModule.Data/Services/CsvPagedDataImporter.cs:58]: Does it moved specially or just unintentionally?

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**